### DISCUSSION TITLE
Column names in translated collection

### DISCUSSION THREAD

**Discussion by lynk** [directus/directus#2698] [Archive]
Hi,

I followed the translation setup on the this page:
https://docs.directus.io/guides/relationships.html#translations

In the article_translation collection I have two fields:
article_title and article_subtitle as text inputs.

In the article collection listing I want to display the article_title and article_subtitles fields as columns, so I am able to sort. 
If I understand the doc correctly this this set in field "Display Template"

![dispaly template](https://user-images.githubusercontent.com/992625/81104675-2704d480-8f13-11ea-9e83-7d9e49473c73.png)

But the columsn do not show up, there is only a column "Translation"

![item_listing](https://user-images.githubusercontent.com/992625/81105790-c080b600-8f14-11ea-9679-771343d5f1f6.png)

**Comment by rijkvanzanten** [comment]
Those two fields in the display template are being rendered inside the _Translations_ column, as that's the field in this collection. All sorting is done on the API, which isn't aware of the display template's output.

**Reply by rijkvanzanten** [reply]
In the next major version, we've included a way to filter and sort on related fields inside the main collection's view, so you will be able to achieve your use case. Until that's out, there's no way to sort on a related field
